src/JSON.jl

```diff
@@ -167,6 +167,18 @@ function _writejson(io::IO, state::State, n::Void)
     Base.print(io, "null")
 end
 
+function _writejson(io::IO, state::State, a::Nullable)
+    if isnull(a)
+        Base.print(io, "null")
+    else
+        _writejson(io, state, get(a))
+    end
+end
+
+function _writejson(io::IO, state::State, c::Char)
```

Comment:
This can probably be made a `lower` method with no performance penalty (since the string has to be allocated anyway). `JSON.lower(c::Char) = string(c)`. The `Nullable` one we can keep as-is, since `lower` would be type-unstable.